templates/login.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Login</title>
+  <link rel="stylesheet" href="/css/styles.css">
+  <link rel="stylesheet" href="https://traquil-ui.netlify.app/styles.css">
+  <script src="https://kit.fontawesome.com/854a67f594.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body id="login-bg">
+  <header>
+    <nav class="nav">
+      <div class="start">
+        <h2 class="theme-heading primary-col">Thrify with me</h2>
+      </div>
+      <ul class="list list-inline f-m">
+        <li class=''>
+          <div class="input-container ">
+            <input placeholder=" " class="input w-full">
+            <label class="input-label f-s">Search</label>
+          </div>
+        </li>
+        <li class='active'>Login</li>
+        <li>Help</li>
+        <li><i class="fas fa-heart primary-col"></i></li>
+        <li><i class="fas fa-shopping-cart"></i></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="grid-three-layout-container">
+    <section class="grid-three-layout-center">
+      <article class="card card-shadow l">
+        <div class="background">
+          <img src="/media/auth/login.jpg" alt="">
+        </div>
+        <div class="card-content stacked">
+          <h2 class="center primary-col f-xl fw-li">Login</h2>
+          <!-- <div class="card-content stacked"> -->
```

Comment:
can use webp instead of jpg or png for better performance , webpage will load faster

Reply:
changed to webp